src/containers/ConceptPage/EditConcept.jsx

```diff
@@ -6,50 +6,66 @@
  *
  */
 
-import React, { Fragment } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { HelmetWithTracker } from '@ndla/tracker';
 import { injectT } from '@ndla/i18n';
 import ConceptForm from './components/ConceptForm';
 import { useFetchConceptData } from '../FormikForm/formikConceptHooks';
 import { LicensesArrayOf } from '../../shapes';
+import { useTranslateConceptForm } from '../FormikForm/translateFormHooks';
+
+import Spinner from '../../components/Spinner';
 
 const EditConcept = ({
   conceptId,
+  isNewlyCreated,
+  licenses,
   selectedLanguage,
   t,
-  isNewlyCreated,
   ...rest
 }) => {
   const {
     concept,
-    updateConcept,
+    fetchSearchTags,
+    fetchStatusStateMachine,
+    loading,
+    setConcept,
     subjects,
+    updateConcept,
     updateConceptAndStatus,
-    fetchStatusStateMachine,
-    fetchSearchTags,
   } = useFetchConceptData(conceptId, selectedLanguage);
 
+  const { translating, translateConcept } = useTranslateConceptForm(
+    concept,
+    setConcept,
+  );
+
   if (!concept) {
     return null;
   }
-
+  if (loading && translating) {
```

Comment:
Quick fix på spinner bugen ;)
```suggestion
  if (loading || translating) {
```

Reply:
jaaa! 🌟